lemonlot-backend/src/main/resources/sql.sql

```diff
@@ -0,0 +1,82 @@
+set search_path to public;
+drop table if exists transactions;
+drop table if exists users;
+drop table if exists cars;
+
+create or replace function update_timestamp() returns trigger
```

Comment:
yeah this update_timestamp function is the only thing I would remove currently but not a big deal since you renamed it to sql.sql anyway